capstone/src/test/java/com/step902020/capstone/CapstoneApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+// package com.step902020.capstone;
```

Comment:
Out of curiosity, does the testd work if uncommented? It seems like it should, although it uses JUnit5, and you've only used JUnit4 so far.

Reply:
It throws errors when uncommented.